types/postcss-media-query-parser/index.d.ts

```diff
@@ -0,0 +1,32 @@
+// Type definitions for postcss-media-query-parser 0.2
+// Project: https://github.com/dryoma/postcss-media-query-parser
+// Definitions by: Remco Haszing <https://github.com/remcohaszing>
+//                 Masafumi Koba <https://github.com/ybiquitous>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+type WalkerCallback = (node: Child, index: number, nodes: Child[]) => boolean | void;
+
+export interface Node {
+    type: string;
+    value: string;
+    after: string;
+    before: string;
+    sourceIndex: number;
+    nodes?: Child[];
```

Comment:
```suggestion
    nodes?: Child[] | undefined;
```

For support for `exactOptionalPropertyTypes`.